lib/internal/bootstrap/switches/does_not_own_process_state.js

```diff
@@ -1,11 +1,18 @@
 'use strict';
 
 const credentials = internalBinding('credentials');
+const rawMethods = internalBinding('process_methods');
+const {
+  unavailable
+} = require('internal/process/worker_thread_only');
+
+process.abort = unavailable('process.abort()');
+process.chdir = unavailable('process.chdir()');
+process.umask = wrappedUmask;
+process.cwd = rawMethods.cwd;
 
 if (credentials.implementsPosixCredentials) {
   // TODO: this should be detached from ERR_WORKER_UNSUPPORTED_OPERATION
```

Comment:
Can you move this comment to where `unavailable` is imported?

Reply:
Sure, done – although to be honest, I’m not exactly sure what `detached` means here.

Reply:
I probably should've used `separated` - because the error message of `unavailable` reads: `%s is not supported in workers`, and in this context it should've been something like `%s is not supported because this Node.js instance does not own the process state`